src/language/branches.md

```diff
@@ -108,11 +116,11 @@ For an empty case that does not fall through, use [`break`][break] for its body.
 
 ### Switch expressions
 
-Switch expressions are similar to switch statements, but you can use them anywhere
-you can use an expression. They produce a value based on the expression body of
-whichever case matches.
+A `switch` expression is similar to a `switch` statement, but you can use them
```

Comment:
**suggestion:** Active voice.
```suggestion
A `switch` expression works like a `switch` statement, but you can use them
```

Reply:
Gonna skip, the difference is more in how they work. Their similarities are also in how they work, but I just don't want to start with that.